ddp-workspace/projects/toolkit-prion/src/lib/components/footer/prionFooter.component.ts

```diff
@@ -22,19 +22,20 @@ import { FooterComponent, ToolkitConfigurationService } from "toolkit";
           <a [href]="'Toolkit.Footer.ContactMailTo' | translate" class="Footer-link NoPadding" translate>Toolkit.Footer.ContactEmail</a>
         </p>
       </div>
-      <div class="col-lg-6 col-md-4 col-sm-8 col-xs-12 Footer-links">
```

Comment:
I think the footer changes must be from a different ticket

Reply:
Done in 9d1d065